### DISCUSSION TITLE
Github Student pack

### DISCUSSION THREAD

**Discussion by snkvrt** [community/community#51206] [GitHub Education]
Hello,

I'm trying to subscribe to the Github student pack but event by giving my proof of enrollment they refuse the demand.
Is it because my proof of enrollment is in french? because I don't know if I can get an english version one...

I tryed to fill my user profil as I can but besides  name, last name, birthday, location what should I fill more ? 

thanks in advance

**Comment by SmashedFrenzy16** [comment]
Hi @RoiJupiter 

Yes, you do need your proof in English. In order to get this, you could contact your institution for an English version. If you need more help on the matter, don't hesitate to contact GitHub Support as they might be able to sort out the issue if your institution cannot provide an English version of your proof of enrolment.

Hope this helps!
